src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -65,24 +134,10 @@ void wrapUp(GitHub root) {
      *      check_run event</a>
      * @see <a href="https://docs.github.com/en/rest/reference/checks#check-runs">Check Runs</a>
      */
-    @SuppressFBWarnings(
-            value = { "UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", "UWF_UNWRITTEN_FIELD", "NP_UNWRITTEN_FIELD" },
-            justification = "JSON API")
     public static class CheckRun extends GHEventPayload {
-        private String action;
         private int number;
         private GHCheckRun checkRun;
         private GHRequestedAction requestedAction;
-        private GHRepository repository;
-
-        /**
-         * Gets action.
-         *
-         * @return the action
-         */
-        public String getAction() {
```

Comment:
Note: all methods that were moved to a superclass were covered with tests `GHEventPayloadTest`. Where were not covered - tests enhanced.